### PR TITLE
fix: link issue

### PR DESCRIPTION
**Pull request by poolpitako**

### CODE REVIEW

File: contracts/Strategy.sol [hunk 1]
```diff
@@ -36,13 +36,13 @@ contract Strategy is BaseStrategy {
     bool internal isOriginal = true;
     // max interest rate we can afford to pay for borrowing investment token
     // amount in Ray (1e27 = 100%)
-    uint256 public acceptableCostsRay = 1e27;
+    uint256 public acceptableCostsRay;
```
Comment (poolpitako) [inline contracts/Strategy.sol:39]: I removed all assignments that happen in `setStrategyParams()`

File: contracts/Strategy.sol [hunk 2]
```diff
@@ -74,14 +78,11 @@ contract Strategy is BaseStrategy {
     IProtocolDataProvider internal constant protocolDataProvider =
         IProtocolDataProvider(0x057835Ad21a177dbdd3090bB1CAE03EaCF78Fc6d);
 
-    address internal constant WETH =
-        address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
-    address internal constant AAVE =
-        address(0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9);
+    address internal constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address internal constant AAVE = 0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9;
```
Comment (poolpitako) [inline contracts/Strategy.sol:82]: I am surprised you missed this haha

File: contracts/Strategy.sol [hunk 3]
```diff
@@ -146,8 +147,8 @@ contract Strategy is BaseStrategy {
         acceptableCostsRay = _acceptableCostsRay;
         maxTotalBorrowIT = _maxTotalBorrowIT;
         referral = _aaveReferral;
-        _setIsInvestmentTokenIncentivised(_isInvestmentTokenIncentivised);
-        _setIsWantIncentivised(_isWantIncentivised);
```
Comment (poolpitako) [inline contracts/Strategy.sol:150]: I removed these setters. If the incentives change, we clone.

File: contracts/Strategy.sol [hunk 4]
```diff
@@ -801,15 +801,6 @@ contract Strategy is BaseStrategy {
...
         // Hack to avoid the stack too deep compiler error.
         SupportStructs.CalcMaxDebtLocalVars memory vars;
-        vars.availableLiquidity = 0;
-        vars.totalStableDebt = 0;
-        vars.totalVariableDebt = 0;
-        vars.totalDebt = 0;
-        vars.utilizationRate = 0;
-        vars.totalLiquidity = 0;
-        vars.targetUtilizationRate = 0;
-        vars.maxProtocolDebt = 0;
```
Comment (poolpitako) [inline contracts/Strategy.sol:811]: by default they are all 0.

File: contracts/Strategy.sol [hunk 5]
```diff
@@ -879,7 +870,10 @@ contract Strategy is BaseStrategy {
             .rayMul(vars.targetUtilizationRate)
             .rayDiv(1e27);
 
-        return (vars.totalDebt, vars.maxProtocolDebt);
+        return (
+            _toETH(vars.totalDebt, address(investmentToken)),
+            _toETH(vars.maxProtocolDebt, address(investmentToken))
```
Comment (poolpitako) [inline contracts/Strategy.sol:875]: bug fix.

File: contracts/Strategy.sol [hunk 6]
```diff
@@ -948,39 +942,38 @@ contract Strategy is BaseStrategy {
             );
     }
 
-    function _setIsWantIncentivised(bool _isWantIncentivised) internal {
-        isWantIncentivised = _isWantIncentivised;
-    }
+    // ----------------- TOKEN CONVERSIONS -----------------
+    function getTokenOutPath(address _token_in, address _token_out)
```
Comment (poolpitako) [inline contracts/Strategy.sol:946]: I changed all path construction logic in the swap methods for a pure function which does the same.

Comment (jmonteer) [reply contracts/Strategy.sol:944]: well done

**Review by jmonteer — APPROVED** [review]

**Comment by jmonteer** [comment]
LGTM

**Pull request merged**